### ISSUE TITLE
Python 2 support with pyriemann>=0.2.7

### ISSUE THREAD

**Issue by jliersch** [pyRiemann/pyRiemann#151]
Hello!

According to the setup.cfg, the package in version 0.2.7 is still intended to work with python 2:

```
[bdist_wheel]
[...]
universal = 1
```

Unfortunately the installation with pip fails for me with python 2.7.18 due to this error:

```
ERROR: Command errored out with exit status 1:
     command: <secret path>/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-jClpfU/pyriemann/setup.py'"'"'; __file__='"'"'/tmp/pip-install-jClpfU/pyriemann/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-jClpfU/pyriemann/pip-egg-info
         cwd: /tmp/pip-install-jClpfU/pyriemann/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-jClpfU/pyriemann/setup.py", line 16, in <module>
        with open('README.md', 'r', encoding="utf8") as fid:
    TypeError: 'encoding' is an invalid keyword argument for this function
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

It's fine for me, if the support for python 2 is dropped with 0.2.7. But I think this should be made consistent.

**Comment by sylvchev** [comment]
Thanks, the support for python 2.7 is dropped since its deprecation. Could you open a PR to push this modification?

**Comment by jliersch** [comment]
I assume there is a way to provide an expressive error message when trying to install pyriemann with python < 3 and think adding such a message is the right thing to do if the support is dropped. Unfortnuately I don't know the best practice how to add such a message yet, but will look into it by the end of the year.

**Comment by jliersch** [comment]
I looked a little deeper into it which resulted in pull request #160, which from my perspective resolves the issue.

**Comment by qbarthelemy** [comment]
Closed by #160.
